### PR TITLE
fix(curriculum): more fixes for rwd beta

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866d28d7ad0de6470505.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-33
 
 # --description--
 
-The flavors and prices are currently stacked on top of each other and centered with their respectable `p` elements. It would be nice if the flavor was on the left and the price was on the right.
+The flavors and prices are currently stacked on top of each other and centered with their respective `p` elements. It would be nice if the flavor was on the left and the price was on the right.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866d28d7ad0de6470505.md:10]: Don't disrespect the p elements :(

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me!

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Comment by ShaunSHamilton** [comment]
Hi @username. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**